src/packages/migrate/src/utils/seed.ts

```diff
@@ -1,47 +1,144 @@
-      return 'npm'
-    }
-    if (usingGlobalYarn) {
-      return 'yarn'
+  // Detect if seed files are next to prisma.schema file
+  const detected = detectSeedFiles(cwd, schemaPath)
+
+  const prismaConfig = await getPrismaConfigFromPackageJson(cwd)
+
+  // If new "seed" config is not set, help user to set it
+  if (!prismaConfig || !prismaConfig.data?.seed) {
+    const packageManager = hasYarn() ? 'yarn add -D' : 'npm i -D'
+
+    // TODO link to docs
+    let message = `${chalk.red(
+      'To configure seeding in your project you need to add a "seed" property in your package.json with the command to execute it:',
+    )}
+
+1. Open the package.json of your project
+`
+
+    if (detected.numberOfSeedFiles === 1) {
+      // Probably was using seed before 2.27.0 and need to add the seed property in package.json
+      message += `2. Add the following example to it:`
+
+      if (detected.js) {
+        message += `
+\`\`\`
+"prisma": {
+  "seed": "node ${detected.seedPath}js"
+}
+\`\`\`
+`
+      } else if (detected.ts) {
+        message += `
+\`\`\`
+"prisma": {
+  "seed": "ts-node ${detected.seedPath}ts"
+}
+\`\`\`
+
+3. Install the required dependencies by running:
+${chalk.green(`${packageManager} ts-node typescript @types/node`)}
+`
+      } else if (detected.sh) {
+        message += `
+\`\`\`
+"prisma": {
+  "seed": "${detected.seedPath}sh"
+}
+\`\`\``
+      }
     } else {
-      false
+      message += `2. Add one of the following example to your package.json:
+
+${chalk.bold('TypeScript:')}
+\`\`\`
+"prisma": {
+  "seed": "ts-node ./prisma/seed.ts"
+}
+\`\`\`
+And install the required dependencies by running:
+${packageManager} ts-node typescript @types/node
+
+${chalk.bold('JavaScript:')}
+\`\`\`
+"prisma": {
+  "seed": "node ./prisma/seed.js"
+}
+\`\`\`
+
+${chalk.bold('Bash:')}
+\`\`\`
+"prisma": {
+  "seed": "./prisma/seed.sh"
+}
+\`\`\``
     }
+
+    throw new Error(message)
+  }
+}
+
+export async function getSeedCommandFromPackageJson(cwd: string) {
+  const prismaConfig = await getPrismaConfigFromPackageJson(cwd)
+  if (!prismaConfig || !prismaConfig.data?.seed) {
+    return null
+  }
+
+  const seedCommandFromPkgJson = prismaConfig.data.seed
+
+  // Validate if seed commad is a string
+  if (typeof seedCommandFromPkgJson !== 'string') {
+    throw new Error(
+      `Provided seed command \`${seedCommandFromPkgJson}\` from \`${path.relative(
+        cwd,
+        prismaConfig.packagePath,
+      )}\` must be of type string`,
+    )
+  }
+
+  if (!seedCommandFromPkgJson) {
+    throw new Error(
+      `Provided seed command \`${seedCommandFromPkgJson}\` from \`${path.relative(
+        cwd,
+        prismaConfig.packagePath,
+      )}\` cannot be empty`,
+    )
+  }
+
+  return seedCommandFromPkgJson
+}
+
+export async function executeSeedCommand(command: string): Promise<boolean> {
+  console.info(`Running seed command \`${chalk.italic(command)}\` ...`)
+  try {
+    const result = await execa.command(command)
```

Comment:
I am wondering about two things on how we execute the commands:

- Are we doing like npm and adding `.npm/bin` to the `PATH`?
- I don't know if we want to, but do we redirect stderr/stdout of the script to the stdout of migrate? It could be useful for debugging seed scripts. At the same time, it could become noisy and messy.